src/runtime/engines/singularity/container.go

```diff
@@ -1580,3 +1583,15 @@ func (c *container) addHostnameMount(system *mount.System) error {
 	}
 	return nil
 }
+
+func (c *container) addActionsMount(system *mount.System) error {
+	hostDir := filepath.Join(buildcfg.SYSCONFDIR, "/singularity/actions")
+	containerDir := "/.singularity.d/actions"
+
+	err := system.Points.AddBind(mount.BindsTag, hostDir, containerDir, syscall.MS_BIND)
```

Comment:
By precaution and to avoid unintentional modification of those files by root from container, I would go for flags `syscall.MS_BIND | syscall.MS_RDONLY | syscall.MS_NOSUID | syscall.MS_NODEV` implying `AddRemount` to apply those flags